stdlib/hashtbl.ml

```diff
@@ -62,7 +62,12 @@ let randomized = Atomic.make randomized_default
 let randomize () = Atomic.set randomized true
 let is_randomized () = Atomic.get randomized
 
+(* CR ocaml 5 runtime:
```

Comment:
We could simplify things a bit by having
```
module Domain = struct
  module DLS = struct
    let new_key ... = ...
    let get ... = ...
  end
end
```
The only `CR ocaml 5 runtime` would
then be to only delete the `Domain`
module.

That being said, I am not positive it is
worth the effort - feel free to resolve
this comment.

Reply:
I lean toward not doing this because there are only two use sites and it's easy enough to see what's going on.